protocol/src/main/java/org/fidoalliance/fdo/protocol/To1ServerService.java

```diff
@@ -25,6 +27,14 @@ protected void doHello(Composite request, Composite reply) {
 
       getStorage().setGuid(guid);
 
+      Date expiryTimeStamp = new SimpleDateFormat("yyyy-MM-DD HH:mm:ss.SSS")
+          .parse(getStorage().getExpiryTimeStamp());
+      Date currentTimeStamp = new SimpleDateFormat("yyyy.MM.dd.HH.mm.ss.SSS")
+          .parse(new SimpleDateFormat("dd.MM.yyyy.HH.mm.ss.SSS").format(new Date()));
```

Comment:
Why is the second format string different?

Reply:
Good catch! Corrected it.